lib/rules/no-else-return.js

```diff
@@ -33,7 +35,56 @@ module.exports = {
          * @returns {void}
          */
         function displayReport(node) {
-            context.report({ node, message: "Unnecessary 'else' after 'return'." });
+            context.report({
+                node,
+                message: "Unnecessary 'else' after 'return'.",
+                fix: fixer => {
+                    const sourceCode = context.getSourceCode();
+                    const alternateStartToken = sourceCode.getFirstToken(node.parent.alternate);
+                    const elseToken = sourceCode.getTokenBefore(alternateStartToken);
+                    const source = sourceCode.getText(node.parent.alternate);
```

Comment:
This line (and the `alternateStartToken` line) is a bit redundant because `node.parent.alternate === node`.